src/ansys/meshing/prime/autogen/connect.py

```diff
@@ -126,13 +126,13 @@ def join_face_zonelets(self, part_id : int, face_zonelet_ids : Iterable[int], wi
         return ConnectResults(model = self._model, json_data = result)
 
     def subtract_volumes(self, part_id : int, target_volumes : Iterable[int], cutter_volumes : Iterable[int], params : SubtractVolumesParams) -> SubtractVolumesResults:
-        """ Subtract cutter volumes from target volumes. Volumes should be computed prior to calling this function.
+        """ Subtract cutter volumes from target volumes. Volumes should be computed prior to calling this function. If multiple parts are being merged to form a single part, then volumes should be computed for each part prior to merging. Use compute_closed_volumes to do so.
 
 
         Parameters
         ----------
         part_id : int
-            Id of a part.
+            Id of part containing target and cutter volumes for subtract operation.
```

Comment:
```suggestion
            ID of the part containing the target and cutter volumes for the subtract operation.
```
While we omit the article at the start of the brief description, we use them inside the description if it improves readability.